src/main/java/net/commments/object/CommentFilter.java

```diff
@@ -0,0 +1,4 @@
+package net.commments.object;
+
+public interface CommentFilter {
```

Comment:
Why did you create this interface? It's not used in the test. Please remove.

Reply:
@extsoft 
it will be used for my next test

Reply:
@lvrub then add in the next test and remove it now.

Reply:
done